#489 Fix causes errors

**Describe the bug**
After #489 fix error detection query gives error (at least when source is PG14)

**To Reproduce**
Steps to reproduce the behavior:
1. Add PG14 source
2. Start service
3. See error

**Expected behavior**
No error and version detection to work


**Desktop (please complete the following information):**
 pgwatch3:
  Version:      master
  DB Schema:    00179
  Git Commit:   cd7c67ce4a2bf6037e1bbe9cf07c6021844d85ea
  Built:        2024-07-25T13:24:09+02:00


**Additional context**
It seems error comes from that query
https://github.com/cybertec-postgresql/pgwatch3/pull/491/files#diff-712163663bb3c92fcb818f61d37d039f0b22f115411f8f05eb798c368e6a03c4R229
https://github.com/cybertec-postgresql/pgwatch3/blob/cd7c67ce4a2bf6037e1bbe9cf07c6021844d85ea/src/reaper/database.go#L229
```
2024-07-31 15:38:05.388 [ERROR] [sql:select /* pgwatch3_generated */
	current_setting('server_version_num')::int / 1_00_00 as ver,
	version(),
	pg_is_in_recovery(),
	current_database()::TEXT,
	system_identifier
FROM
	pg_control_system()] [err:ERROR: syntax error at or near "as" (SQLSTATE 42601)] [name:stmtcache_302ba4639f30b795b51999b9746e054e1830a1f4ae05ce8f] [pid:1622716] [time:953.791µs] Prepare
2024-07-31 15:38:05.388 [ERROR] [sql:select /* pgwatch3_generated */
	current_setting('server_version_num')::int / 1_00_00 as ver,
	version(),
	pg_is_in_recovery(),
	current_database()::TEXT,
	system_identifier
FROM
	pg_control_system()] [args:[]] [err:ERROR: syntax error at or near "as" (SQLSTATE 42601)] [pid:1622716] [time:1.053792ms] Query
2024-07-31 15:38:05.388 [ERROR] could not start metric gathering for DB "Lab_zabbix" due to connection problem: ERROR: syntax error at or near "as" (SQLSTATE 42601)
```

Possible, it should be `current_setting('server_version_num')::int / 10000 as ver,` 
For me it seems a bit odd to divide version like that. If it to visually separate number, then such formatting functionality was added in PG16 https://www.postgresql.org/docs/16/release-16.html#RELEASE-16-DATATYPES 
Some extra documentation: https://www.postgresql.org/docs/16/sql-syntax-lexical.html#SQL-SYNTAX-CONSTANTS-NUMERIC 
Currently PG14 seems to treat it as: `current_setting('server_version_num')::int / 1 as _00_00 as ver`

--- a/src/reaper/database.go
+++ b/src/reaper/database.go
@@ -226,7 +226,7 @@ func GetMonitoredDatabaseSettings(ctx context.Context, dbUnique string, srcType 
 		dbNewSettings.Version = VersionToInt(matches[0])
 	default:
 		sql := `select /* pgwatch3_generated */ 
-	current_setting('server_version_num')::int / 1_00_00 as ver, 
+	current_setting('server_version_num')::int / 10000 as ver, 
 	version(), 
 	pg_is_in_recovery(), 
 	current_database()::TEXT,

